src/components/RenderCounter.tsx

```diff
@@ -6,7 +6,7 @@ type RenderCounterProps = {
   style?: React.CSSProperties;
 };
 
-const RenderCounter = React.memo<RenderCounterProps>(props => {
+const RenderCounter: React.FC<RenderCounterProps> = props => {
```

Comment:
IDK why it was pure as it breaks render cycle. Hope that it will not break anything other